src/Server/TCPHandler.cpp

```diff
@@ -1752,6 +1753,24 @@ void TCPHandler::sendHello()
         nonce.emplace(thread_local_rng());
         writeIntBinary(nonce.value(), *out);
     }
+
+    if (client_tcp_protocol_version >= DBMS_MIN_REVISION_WITH_SERVER_HELLO_EXTRAS)
+    {
+        /// Write newly added fields as one length-prefixed blob to allow proxies to skip them
+        /// without parsing. Needed only for Hello message because it can't use chunked protocol.
```

Comment:
this comment is misleading - handshake is always not chunked - please remove

Reply:
explanation: this is very nice of you, but not necessary - handshake in proxy parser meant to be updated when protocol changes - it's not that important do we need add one string parser or string array parsing for example. So send it as you see fit.